### ISSUE TITLE
[Accessibility]: StackedBarChart progress bar not receiving focus

### ISSUE THREAD

**Issue by rileydanejohnston** [carbon-design-system/carbon-charts#1977]
### Application/Team

billing

### Environment

Browser

### What happened? What did you expect to happen instead?

Original issue - https://github.ibm.com/ibmcloud/billing/issues/3386

Issue description: The interactive progress bar, which is supposed to display tooltip information for each section of the slider, is not receiving keyboard focus.

Recommended fix: The interactive progress bar, which is supposed to display tooltip information for each section of the slider must receive keyboard focus.

User impact: Users relying on keyboard navigation/Screen reader users  are unable to interact with the progress bar or view the tooltip information.

WCAG Mapping: 2.1.1 Keyboard

Level of Compliance: A

Severity: Medium

Test tool: Keyboard_Firefox

Screen print:

![Image](https://github.com/user-attachments/assets/69a011fe-932a-4ef6-8dc6-2457140d74c7)

### What WCAG 2.1 checkpoint does the issue violate?

WCAG Mapping: 2.1.1 Keyboard

### Version

"@carbon/charts-react": "^1.22.18",

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P2

**Comment by theiliad** [comment]
Hi,
We’ve done various rounds of accessibility analysis and design around our charts.

Our accessibility designer at the time was of the opinion that keyboard navigation through charts is not a great idea, since it doesn’t actually paint a mental picture of what the chart is visualizing (e.g. trends, patterns etc…)

That is why the tabular view was designed in charts
